emba.sh

```diff
@@ -14,6 +14,7 @@
 
 # Description:  Main script for load all necessary files and call main function of modules
 
+INVOCATION_PATH="."
```

Comment:
Please split this pull request into two:
1st - cosmetic updates like this (we can accept this)
2nd - grep output pull request (we should test this)